pipeline/sources/pleiades/mapper.py

```diff
@@ -0,0 +1,205 @@
+        same_as_match = re.search(r'owl:sameAs\s+(.+?);', ttl_section, re.DOTALL)
+        same_as_matches = []
+        if same_as_match:
+            same_as_section = same_as_match.group(1)
+            # Extract all URIs from this section
+            same_as_matches = re.findall(r'<([^>]+)>', same_as_section)
+        
+        # Create the Type record
+        top = model.Type(ident=uri)
+        
+        # Add primary name
+        primary_name = vocab.PrimaryName(content=label)
+        primary_name.language = model.Language(ident="http://vocab.getty.edu/aat/300388277")  # English
+        top.identified_by = primary_name
+        
+        # Add description if present
+        if description:
+            desc = vocab.Description(content=description)
+            desc.language = model.Language(ident="http://vocab.getty.edu/aat/300388277")  # English
+            top.referred_to_by = desc
+        
+        # Add equivalents from owl:sameAs statements
+        for same_as_uri in same_as_matches:
+            # Skip self-references
+            if same_as_uri != uri:
+                top.equivalent = model.Type(ident=same_as_uri)
+            
+        
+        data = model.factory.toJSON(top)
+        return {"identifier": concept_id, "data": data, "source": "pleiades"}
+
+    def parse_place(self, record):
+        rec = record["data"]
+        recid = record["identifier"]
+
+        # Handle places
+        top = model.Place(ident=rec["uri"])
+        # Collect all available names
+        all_names = []
+        title = rec.get("title")
+        if title:
+            all_names.append({"content": title, "language": "en", "source": "title"})
+        
+        # Add names from the names array
+        names = rec.get("names", [])
+        for n in names:
+            if "attested" in n and n["attested"]:
+                lang = n.get("language", "en")
+                all_names.append({"content": n["attested"], "language": lang, "source": "names"})
+        
+        if not all_names:
+            return None
+        
+        # Assign first name as primary
+        primary_name_data = all_names[0]
+        primary_name = vocab.PrimaryName(content=primary_name_data["content"])
+        if primary_name_data["language"] == "en":
+            primary_name.language = model.Language(ident="http://vocab.getty.edu/aat/300388277")  # English
+        else:
+            primary_name.language = model.Language(label=primary_name_data["language"])
+        top.identified_by = primary_name
+        
+        # Add remaining names as alternate names
+        for name_data in all_names[1:]:
+            alt_name = vocab.AlternateName(content=name_data["content"])
+            if name_data["language"] == "en":
+                alt_name.language = model.Language(ident="http://vocab.getty.edu/aat/300388277")  # English
+            else:
+                alt_name.language = model.Language(label=name_data["language"])
```

Comment:
Ditto -- no point having a language without a URI, but we have a good alignment table.